examples/nextjs/pages/vue.tsx

```diff
@@ -0,0 +1,45 @@
+import { NextPage } from 'next';
+import { useEffect } from 'react';
+import { initializeSwapWidget } from '@skip-go/widget';
+
+const VuePage: NextPage = () => {
+  initializeSwapWidget();
+
+  useEffect(() => {
+    const { createApp } = require('vue/dist/vue.esm-bundler.js');
+    const app = createApp({
+      setup() {
+        const colors = JSON.stringify({
+          primary: '#FF4FFF',
+        });
+        const defaultRoute = JSON.stringify({
+          srcChainID: 'osmosis-1',
+          srcAssetDenom:
+            'ibc/1480b8fd20ad5fcae81ea87584d269547dd4d436843c1d20f15e00eb64743ef4',
+        });
+
+        return {
+          colors,
+          defaultRoute,
+        };
+      },
+      template: `
+        <div style="width:450px;height:820px;">
+          <skip-widget :colors="colors" :default-route="defaultRoute"></skip-widget>
+        </div>
+      `,
+    });
+
+    (app.config.compilerOptions.isCustomElement = (tag: string) =>
+      ['skip-widget'].includes(tag)),
+      app.mount('#vue');
+
+    return () => {
+      app.unmount();
+    };
+  }, []);
```

Comment:
ideally I would just have a nuxtjs package in `examples/nuxtjs` but I'm running into issues just setting that up because there's issues between `nuxtjs` install and `yarn workspaces`